Zend/zend_inheritance.c

```diff
@@ -253,7 +253,10 @@ static int zend_do_perform_arg_type_hint_check(const zend_function *fe, zend_arg
 	}
 
 	if (!ZEND_TYPE_IS_SET(proto_arg_info->type)) {
-		/* Child defines a type, but parent doesn't, violates LSP */
+		/* Child defines a type, but parent doesn't, violates LSP, unless it's a mixed type */
+		if (ZEND_TYPE_CODE(fe_arg_info->type) == IS_MIXED) {
+			return 1;
+		}
 		return 0;
```

Comment:
I believe this can be simplified:
```suggestion
	    return ZEND_TYPE_CODE(fe_arg_info->type) == IS_MIXED;
```